optimize/client/scripts/dependencies.js

```diff
@@ -34,7 +33,7 @@ const lines = toPairs(dependencies)
   }, []);
 
 function getLibraryLink(name, rawUrl, repository) {
-  const parsedUrl = rawUrl ? url.parse(rawUrl) : {};
+  const parsedUrl = rawUrl ? new URL(rawUrl.startsWith('http') ? rawUrl : `https://${rawUrl}`) : {};
```

Comment:
and what if the url starts with `https`? then still the `startsWith('http')` will be true. Do you think it can cause issues? What was the reason for this change? Didn't the `url.parse` method work?